executor/aggfuncs/builder.go

```diff
@@ -445,6 +447,29 @@ func buildVarPop(aggFuncDesc *aggregation.AggFuncDesc, ordinal int) AggFunc {
 	}
 }
 
+// buildStdDevPop builds the AggFunc implementation for function "STD()/STDDEV()/STDDEV_POP()"
+func buildStdDevPop(aggFuncDesc *aggregation.AggFuncDesc, ordinal int) AggFunc {
+	base := baseStdDevPopAggFunc{
+		varPop4Float64{
+			baseVarPopAggFunc{
+				baseAggFunc{
+					args:    aggFuncDesc.Args,
+					ordinal: ordinal,
+				},
+			},
+		},
+	}
+	switch aggFuncDesc.Mode {
+	case aggregation.DedupMode:
+		return nil
+	default:
+		if aggFuncDesc.HasDistinct {
+			return &stdDevPop4DistinctFloat64{base}
```

Comment:
This function is not tested, please add some test cases.

Reply:
The unit tests contains the distinct attribute test case. Should I add more test cases?
https://github.com/pingcap/tidb/blob/master/executor/aggfuncs/aggfunc_test.go#L397

Reply:
Sure, I would like to see test cases that are directly against stddev distinct, please add some of these tests.

Reply:
Could you give me a example or tell me which function has test cases about distinct attribute? I'm not sure how to add test cases that against stddev distinct.

Reply:
Have you tested something like select stddev(distinct a) from t? Sorry but I am on commute right now, I would like to help on this later or tomorrow if you still have questions. (I might be wrong on this, but the implementation of distinct stddev seems questionable to me)

Reply:
I've tested select stddev(distinct xx) from t manually, but seems to find a bug.
I created a issue: https://github.com/pingcap/tidb/issues/19499.
It seems var_pop/stdev func couldn't contains distinct in it in MySQL.

Reply:
Good job! Then perhaps we could remove the `stdDevPop4DistinctFloat64` here, because we don't need to support it.